src/GovUk.Education.ExploreEducationStatistics.Content.Model/Release.cs

```diff
@@ -64,6 +64,8 @@ public string ReleaseName
 
         public List<Update> Updates { get; set; }
 
+        public List<ReleaseStatus> ReleaseStatuses { get; set; }
```

Comment:
I think the task to remove the old InternalReleaseNote from Release is fine to be done separately but it wants doing straightafter otherwise it's going to be confusing as it's redundant and not necessarily the latest note.

Reply:
I've created a ticket to do this and its in selected for development - I'll do it as soon as this gets merged